sktime/utils/validation/tests/test_forecasting.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Test forecasting module."""
 
 __author__ = ["Markus Löning"]
```

Comment:
while you´re here, kindly change this to GitHub Id (`mloning`)

Reply:
changed